test/regress/cli/run_regression.py

```diff
@@ -37,85 +37,101 @@ class Color:
     ENDC = "\033[0m"
 
 
+def print_info(msg):
+    print(Color.BLUE + "▶ " + msg + Color.ENDC)
+
+
+def print_ok(msg):
+    print(Color.GREEN + "✓ " + msg + Color.ENDC)
+
+
+def print_error(err):
+    print(Color.RED + "✖ " + err + Color.ENDC)
+
+
 class Tester:
-    def __init__(self):
-        pass
+
+    def __init__(self, name):
+        self.name = name
 
     def applies(self, benchmark_info):
         return True
 
-    def check_exit_status(self, name, expected_exit_status, exit_status, output, error, flags):
+    def check_exit_status(self, expected_exit_status, exit_status, output,
+                          error, flags):
         if exit_status == STATUS_TIMEOUT:
-            print("Timeout ({}) - Flags: {}".format(name, flags))
+            print_error("Timeout")
             return EXIT_SKIP if g_args.skip_timeout else EXIT_FAILURE
         elif exit_status != expected_exit_status:
-            print('not ok ({}) - Expected exit status "{}" but got "{}" - Flags: {}'.format(
-                name, expected_exit_status, exit_status, flags))
-            print()
-            print_outputs(output, error)
+            print_error(
+                'Unexpected exit status: expected "{}" but got "{}"'.format(
+                    expected_exit_status, exit_status))
             return EXIT_FAILURE
-        return None
+        return EXIT_OK
 
     def run(self, benchmark_info):
+        """Prints information about the test that is about to run and then
+        calls the internal run method to execute the test."""
+
+        print()
+        print_info(self.name)
+        print("  Flags: {}".format(benchmark_info.command_line_args))
```

Comment:
Commands are printed before they execute. So, I wonder if we need this line anymore...

Reply:
Yeah, I see where you are coming from. I'd like to keep it for now, though, because if you have multiple `COMMAND-LINE` directives, it is useful to quickly see which one is failing.